odoo/addons/base/data/res_country_data.xml

```diff
@@ -1410,7 +1410,7 @@
             <field name="name">Qatar</field>
             <field name="code">qa</field>
             <field file="base/static/img/country_flags/qa.png" name="image" type="base64" />
-            <field eval="'%(street)s\n%(street2)s\n%(state_name)s\n%(country_name)s'" name="address_format" />
+            <field eval="'%(street)s\n%(street2)s\n%(city)s %(state_name)s\n%(country_name)s'" name="address_format" />
```

Comment:
@mpa-odoo  remove state_name for Qatar because address formate is street_city_country